scripts/socket-helpers/utils.ts

```diff
@@ -0,0 +1,67 @@
+import { Contract, utils } from "ethers";
+import { defaultAbiCoder } from "ethers/lib/utils";
+import PlugABI from "@socket.tech/dl-core/artifacts/abi/IPlug.json";
+
+export type TxData = {
+  from: string;
+  to: string;
+  data: string;
+};
+
+export type Inputs = {
+  amount: string;
+  receiver: string;
+  executionData: string;
+  connectorPlug: string;
+};
+
+export type ChainDetails = {
+  srcChainSlug: number;
+  dstChainSlug: number;
+};
```

Comment:
why havent you used ChainSlug type here instead of number? And typecasting to chainslug in getEstimatedGasLimit immediately after input?

Reply:
Also ChainDetails name seems like it will have details of a chain i.e rpc, explorer etc
can we just put these in Inputs as well?

Reply:
fixed

Reply:
inputs were the bridge related details like amount, receiver etc while chain details is separate